bouncer/blacklist/tests.py

```diff
@@ -1,32 +1,50 @@
 from django.test import TestCase
 from blacklist import models
-from blacklist.brain import is_ip_blacklisted, is_email_blacklisted
+from blacklist.brain import (
+    is_ip_blacklisted,
+    is_email_blacklisted,
+    is_email_host_blacklisted,
+)
 
 
 class BrainTests(TestCase):
     @classmethod
     def setUpClass(cls):
         super(BrainTests, cls).setUpClass()
-        cls.blacklisted_ip = "1.2.3.4"
+        cls.lower_case_blacklisted_ip = "2001:0:3238:dfe1:63::fefb"
+        cls.upper_case_blacklisted_ip = "2001:0:3238:DFE1:63::FEFB"
```

Comment:
Better use something like `cls.upper_case_blacklisted_ip = cls.lower_case_blacklisted_ip.upper()`. It's easier to read and more robust than human copy pasting and capitalization 😄